draft-ietf-quic-transport.md

```diff
@@ -2483,8 +2489,8 @@ after a frame bearing the FIN flag is sent.
 
 A stream is "half-closed (remote)" when the stream is no longer being used by
 the peer to send any data.  An endpoint will have either received all data that
-a peer has sent or will have received a RST_STREAM frame and discarded any
-received data.
+a peer has sent, will have sent an UNI flag on the stream, or will have
```

Comment:
Possibly you or Buck have suggestions on how the HTTP mapping should change?  Except for server push, I don't think it needs to change at all, and that change should be as minor as specifying that server push streams should be initiated with the unidirectional bit set to true.